versions/versions.md

```diff
@@ -0,0 +1,18 @@
+# Versions
+
+This document is meant to be a convenient reference to assist developers in checking out the desired old version of `Unlock.sol` or `PublicLock.sol` for reference/debugging purposes.
+The commit hashes represent the final state of the given contract prior to being upgraded or, in the case of the Lock contract, having the version number bumped.
+
+Use the command `$ git checkout <commit_hash_here>`to view the unlock repository at this point in time.
+
+After the upgrade to `Unlock.sol` V1, we'll need to modify our test suite to use the artifacts from this directory when testing Unlock V0, so: `const Unlock = artifacts.require('../../versions/Unlock_V0.json')`
+
+Note: It's possible that the artifacts.require syntax won't work for this use case. If this is the case, we'll need to look at using truffle-contract instead: https://github.com/trufflesuite/truffle/tree/master/packages/truffle-contract
+
+## Unlock.sol
+
+- `Unlock.sol` (V0) commit: <add commit# prior to upgrading to UnlockV1.sol>
+
+## PublicLock.sol
+
+- `PublicLock.sol` (V0) commit: 9ee8a2dcd037cbab747bfb0b5cf28e2b7b38360e
```

Comment:
This commit seems to be from after the v0 tag.  Shouldn't we use exactly what was there when deployed?

Reply:
You are right. Fixing this now.

Reply:
Done.